### PR TITLE
[14.0][MIG] account_mass_reconcile_as_job: Migration to 14.0

### PR DESCRIPTION
**Pull request by anothingguy**
Depends on:

- account_mass_reconcile: #371

### CODE REVIEW

File: account_mass_reconcile_as_job/data/queue_job_channel_data.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<odoo noupdate="1">
+    <record model="queue.job.channel" id="channel_mass_reconcile">
+        <field name="name">mass.reconcile.job</field>
```
Comment (sebalix) [inline account_mass_reconcile_as_job/data/queue_job_channel_data.xml:3]: The previous channel used with the `job` decorator was named `mass_reconcile`, better to keep it for existing instance.

**Review by sebalix — COMMENTED** [review]
One comment, LG otherwise

**Comment by sebalix** [comment]
@anothingguy I open a PR on yours to add a feature (to keep in a separate commit): https://github.com/anothingguy/account-reconcile/pull/1

**Review by grindtildeath — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]
Code review. LG

**Pull request merged**